api/v1alpha5/zz_generated.conversion.go

```diff
@@ -24,6 +24,7 @@ package v1alpha5
 import (
 	unsafe "unsafe"
 
+	v1alpha4 "./api/v1alpha4"
```

Comment:
One way to make the compiler happy would be to change this in the following way:
```suggestion
	v1alpha4 "sigs.k8s.io/cluster-api-provider-openstack/api/v1alpha4"
```
Any clue how I can make code-gen understand this?